Text/CustomEntitySearch/README.md

```diff
@@ -109,10 +115,18 @@ Here's a sample skill definition for this example (inputs and outputs should be
         {
             "name": "words",
             "source": "/document/merged_content/*"
+        },
+		{
+            "name": "languageCode",
+            "source": "/document/merged_content/*"
```

Comment:
This path doesn't look right. @jadrefke can you take another look at this skill definition and advise?

Reply:
default output location for [language detection skill](https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-language-detection) language code is /document/languageCode

---
In reply to: [305524700](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r305524700) [](ancestors = 305524700)

Reply:
text's source should probably be /document/content

words source would likely be something custom.. Another option would be to include the list of words in a custom webApiHeader... then you don't need to grab it from each individual document

---
In reply to: [305533021](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r305533021) [](ancestors = 305533021,305524700)

Reply:
For now, I'll keep words as is wrt the skill definition so as to maintain a general webAPIHeader. For prod dev, I think it would definitely be better to have a custom one though